content/docs/tech-insights/track-dependency-versions/index.md

```diff
@@ -0,0 +1,273 @@
+
+    ![About Data Source section displayed](./react/React_Applies_To_Section.png)
+
+8. Press **SAVE**.
+
+## Visualise the distribution of React versions being used
+Data Source comes with a built-in visualisation panel which lets us get an overview of the React versions being used. When viewing a Data Source, expand the “Facts visualization” section to see it.
+
+![Data Visualisation results displayed](./react/React_Data_Visualization_Section.png)
+
+This chart tells us:
+
+1. 17% of the Components that this Data Source targets are using version `17.0.59`.
+2. 17% are using version `15.4.0`.
+3. 67% of Components do not have a React version at all. This is the **Empty Results** section.
+
+In the next section, we will create a Check which can show a pass or fail result to app dev teams to tell them if they need to upgrade their React version.
+
+## Create a check that shows which software is not using a supported React version
+
+We have a data source telling us what React versions are present in our components, but it also includes non-React components in the results. Let's omit the non-React components from the results by creating a check.
+
+### Create a check
+
+The builtin "Repository Files Data Source" gives us all the file paths of a component's repository. We can use this to determine which components are using React. We can also now extract the React version from those files.
+
+Let’s write a check to combine both of these properties.
+
+1. Visit Tech Insights, select the **Checks** tab, and press **ADD CHECK**.
+
+    ![Add Check button](./Add_Check.png)
+
+2. Enter a descriptive **Name** and **Description**.
+
+   ![About section input fields](./react/React_Check_About_Section.png)
+
+3. Create a compound check:
+    1. Set **Type** to `Or`.
+    2. Set the first condition to:
+        | Field Name | Value |
+        | --- | --- |
+        | Data Source | GitHub repository file list |
+        | Fact | List of files |
+        | Fact operator | Does not contain |
+        | Value | package.json |
+
+    2. Press **+ADD CONDITION** and set the second condition to:
+        | Field Name | Value |
+        | --- | --- |
+        | Data Source | Package JSON information |
+        | Fact | React version |
+        | Fact operator | Greater than semantic version |
+        | Value | 16.0.0 |
+    3. Press **+ADD CONDITION** and set the third condition to:
+        | Field Name | Value |
+        | --- | --- |
+        | Data Source | Package JSON information |
+        | Fact | Has React dependency |
+        | Fact operator | Is False |
+
+    ![Check Condition section fields](./react/React_Check_Condition_Section.png)
+
+4. Press **DRY RUN**
+
+5. (Optional) Add a URL to documentation outlining the steps to update the React version in the component.
```

Comment:
```suggestion
5. (Optional) Add a URL to documentation outlining the steps to update the React version in the component. You can also add a link to a scaffolder template that automatically achieves this fix.
```

Reply:
The original purpose of the `fix link` functionality was to be able to direct users to scaffolder for these fixes